src/xrAICore/Navigation/level_graph_inline.h

```diff
@@ -80,19 +80,23 @@ ICF const Fvector& CLevelGraph::vertex_position(
     return (dest_position = vertex_position(source_position));
 }
 
-// XXX: This method is WAY to large to inline.
-IC const CLevelGraph::CPosition& CLevelGraph::vertex_position(CPosition& dest_position, const Fvector& source_position) const
+IC int CLevelGraph::calculate_packed_xz(const float x, const float z) const
 {
     const auto [box_x, box_y, box_z] = header().box().vMin;
     const auto cell_size = header().cell_size();
+    const int packed_xz = iFloor((x - box_x) / cell_size + .5f) * m_row_length +
+		iFloor((z - box_z) / cell_size + .5f);
 
-    VERIFY(iFloor((source_position.z - box_z) / cell_size + .5f) < (int)m_row_length);
```

Comment:
This VERIFY was removed. Needs to be returned.

Reply:
Ops! ...returned.